experiment-setup/execution/execution.py

```diff
@@ -7,53 +7,61 @@
 from otii_config import create_otii_app, configure_multimeter
 from data_collection import collect_data, generate_output, save_data, save_sequential_time
 from orchestration import clean_database, manage_server_docker_service, trigger_clients
-from client.host_sequence.scenario_api import run_api_seq_scenario
-from client.host_sequence.scenario_page import run_page_seq_scenario
+from host_sequence.scenario_api import run_api_seq_scenario
+from host_sequence.scenario_page import run_page_seq_scenario
 
 # Check the static addresses when preparing the setup!
-SERVER_URL = "http://10.0.0.4:5000"
-CLIENT_1_URL = "http://10.0.0.3:5001/trigger"
-CLIENT_2_URL = "http://10.0.0.2:5001/trigger"
-CLIENT_3_URL = "http://10.0.0.5:5001/trigger"
+SERVER_URL = "http://10.7.7.144:5000"
+CLIENT_1_URL = "http://10.7.7.199:5001/trigger"
+CLIENT_2_URL = "http://10.7.7.178:5001/trigger"
+CLIENT_3_URL = "http://10.7.7.145:5001/trigger"
 
-SERVICES = ["rust-actix", "python-flask"]
+BASE_COMPOSE_FILES_LOCATION = '/media/mmcblk0p2/setup/compose_files/'
+COOLDOWN = 30
 
-async def main(otii_project, device, out_path, service, run_mode="standard"):
-    print("Clearing DB on remote server...", flush=True)
-
-    reset = clean_database()
+SERVICES = {#"rust-actix": BASE_COMPOSE_FILES_LOCATION + 'rust-actix-compose-prod.yml'}
+            "python-flask": BASE_COMPOSE_FILES_LOCATION + 'python-flask-compose-prod.yml'}
 
+async def main(otii_project, device, out_path, service, run_mode):
+    client_urls = [CLIENT_1_URL, CLIENT_2_URL, CLIENT_3_URL]
+    time_seq_api_df = None
+    time_seq_page_df = None
+    print("Clearing DB on remote server...", flush=True)
+    reset = clean_database(run_mode == "berries")
     if reset:
-        print("Starting scenario on three clients...", flush=True)
+        print("DB cleared...", flush=True)
+        print(f'Starting experiment - scenario: {run_mode}, service: {service}...', flush=True)
         start_time = datetime.now()
-
         otii_project.start_recording()
-        if run_mode == "standard":
-            client_urls = [CLIENT_1_URL, CLIENT_2_URL, CLIENT_3_URL]
-            result_clients_trigger = trigger_clients(client_urls)
+        if run_mode == "berries":
+            result_clients_trigger = await trigger_clients(client_urls)
             print(result_clients_trigger, flush=True)
         elif run_mode == "sequential":
-            time_seq_api_df = await asyncio.create_task(run_api_seq_scenario(service, start_time))
-            time_seq_page_df = await asyncio.create_task(run_page_seq_scenario(service, start_time))
-
+            time_seq_api_df = run_api_seq_scenario(service, start_time)
+            reset = clean_database(False)
+            time_seq_page_df = run_page_seq_scenario(service, start_time)
```

Comment:
Maybe we can concat the two dataframe here so we just have one:
`combined_df = pd.concat(dataframe, ignore_index=True)`